fission-core/library/Fission/URL/Subdomain/Types.hs

```diff
@@ -72,226 +67,5 @@ instance MimeUnrender PlainText Subdomain where
-
-      else do
-        adjectives <- sequence (elements <$> take 3 generators)
-        noun       <- elements nouns
-        return . Subdomain . Text.intercalate "-" $ adjectives <> [noun]
-
-opinions :: [Text]
-opinions =
-  [ "amazing"
-  , "beautiful"
-  , "ugly"
-  , "quick"
-  , "wonderful"
-  , "awesome"
-  , "sweet"
-  , "tubular"
-  , "eager"
-  , "magnificient"
-  , "nice"
-  , "lively"
-  , "bewildered"
-  , "fierce"
-  , "jolly"
-  , "victorious"
-  , "calm"
-  , "brave"
-  , "proud"
-  , "fancy"
-  , "skinny"
-  , "bald"
-  , "elegant"
-  , "muscular"
-  , "rich"
-  , "tasty"
-  , "super"
-  , "brainy"
-  , "infantile"
-  , "juvenile"
-  , "spry"
-  , "playful"
-  , "loyal"
-  , "vicious"
-  , "cute"
-  , "benevolent"
-  , "malevolent"
-  , "universal"
-  , "narcissistic"
-  , "wise"
-  , "stupid"
-  , "obtuse"
-  , "fun"
-  , "charming"
-  , "wicked"
-  , "gnarly"
-  ]
-
-sizes :: [Text]
-sizes =
-  [ "big"
-  , "small"
-  , "little"
-  , "huge"
-  , "enormous"
-  , "petite"
-  , "tall"
-  , "short"
-  , "tiny"
-  , "colossal"
-  , "long"
-  , "short"
-  , "long"
-  , "gigantic"
-  ]
-
-ages :: [Text]
-ages =
-  [ "old"
-  , "young"
-  , "ancient"
-  , "elderly"
-  , "senior"
-  , "junior"
-  ]
-
-shapes :: [Text]
-shapes =
-  [ "wide"
-  , "narrow"
-  , "round"
-  , "triangular"
-  , "square"
-  , "flat"
-  , "thin"
-  , "thick"
-  , "skinny"
-  , "aerodynamic"
-  , "oval"
-  , "angular"
-  ]
-
-colours :: [Text]
-colours =
-  [ "red"
-  , "green"
-  , "blue"
-  , "yellow"
-  , "orange"
-  , "teal"
-  , "white"
-  , "black"
-  , "purple"
-  , "pink"
-  , "magenta"
-  , "cyan"
-  , "brown"
-  , "maroon"
-  , "aquamarine"
-  , "fuchsia"
-  , "crimson"
-  , "scarlet"
-  , "turquoise"
-  ]
-
-materials :: [Text]
-materials =
-  [ "glass"
-  , "wooden"
-  , "metalic"
-  , "leather"
-  , "polyester"
-  , "silk"
-  , "velvet"
-  , "nylon"
-  , "stone"
-  , "diamond"
-  , "plastic"
-  , "tin"
-  , "carbon"
-  , "cardboard"
-  , "paper"
-  , "sand"
-  , "plaster"
-  , "silicon"
-  , "canvas"
-  , "wool"
-  , "cotton"
-  , "marble"
-  ]
-
-nouns :: [Text]
-nouns =
-  [ "dragon"
-  , "unicorn"
-  , "mermaid"
-  , "fairy"
-  , "werewolf"
-  , "sphinx"
-  , "yeti"
-  , "griffin"
-  , "wolf"
-  , "crow"
-  , "centaur"
-  , "imp"
-  , "ghoul"
-  , "pixie"
-  , "gnome"
-  , "wizard"
-  , "witch"
-  , "mage"
-  , "troll"
-  , "cat"
-  , "dog"
-  , "snake"
-  , "lion"
-  , "monkey"
-  , "tiger"
-  , "fish"
-  , "crab"
-  , "shark"
-  , "salmon"
-  , "tuna"
-  , "horse"
-  , "turtle"
-  , "dolphin"
-  , "deer"
-  , "leopard"
-  , "bear"
-  , "frog"
-  , "llama"
-  , "penguin"
-  , "pig"
-  , "eagle"
-  , "bat"
-  , "vampire"
-  , "dinosaur"
-  , "whale"
-  , "king"
-  , "queen"
-  , "jester"
-  , "butterfly"
-  , "tulip"
-  , "polar-bear"
-  , "cactus"
-  , "hero"
-  , "knight"
-  , "hippogriff"
-  , "elf"
-  , "beast"
-  , "sprite"
-  , "alien"
-  , "ghost"
-  , "martian"
-  , "princess"
-  , "prince"
-  , "chef"
-  , "barista"
-  , "ufo"
-  ]
```

Comment:
Moved this to `AppName`, since that's what it is. Subdomain is a much more general thing, and we may want to process subdomains on user domains which don't have the same `${appName}.fission.app` format